pkg/webui/components/link/index.js

```diff
@@ -118,6 +119,69 @@ Link.defaultProps = {
   titleValues: undefined,
 }
 
+const DocLink = function(props) {
+  const {
+    className,
+    name,
+    title,
+    titleValues,
+    id,
+    path,
+    children,
+    showVisited,
+    intl,
+    secondary,
+    primary,
+    disabled,
+  } = props
+
+  const classNames = classnames(style.link, className, {
+    [style.linkVisited]: showVisited,
+    [style.primary]: primary,
+    [style.secondary]: secondary,
+  })
+  if (disabled) {
+    return <span className={classnames(classNames, style.disabled)}>{children}</span>
+  }
+  const link = generateDocUrl(path)
+  const formattedTitle = formatTitle(title, titleValues, intl.formatMessage)
+
+  return (
+    <a className={classNames} title={formattedTitle} id={id} href={link} target="blank" name={name}>
+      {children}
+    </a>
+  )
+}
+
+DocLink.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  disabled: PropTypes.bool,
+  id: PropTypes.string,
+  intl: PropTypes.shape({
+    formatMessage: PropTypes.func,
+  }).isRequired,
+  name: PropTypes.string,
+  path: PropTypes.string.isRequired,
+  primary: PropTypes.bool,
+  secondary: PropTypes.bool,
+  showVisited: PropTypes.bool,
+  title: PropTypes.string,
+  titleValues: PropTypes.shape({}),
```

Comment:
Better to spread the prop types from the other link component. There's too much duplication.

```js
DocLink.propTypes = {
   ...Link.propTypes,
   …
}
```